.github/ISSUE_TEMPLATE/functional_bug.yml

```diff
@@ -0,0 +1,89 @@
+# yamllint disable rule:line-length
+---
+body:
+  - attributes:
+      description: >-
```

Comment:
I see that most of the time you use `>` for multiline strings, but there are some times you use `>-` (which means "don't append a linebreak at the end", if I'm not mistaken). Is there a reason for that?

Reply:
Not really. Emacs just automatically turns `>` into `>-`.

But GitHub's issue forms renderer doesn't even distinguish between `>` (multiline string, drop newlines) and `|` (multiline string, retain newlines); it always behaves as if using `|`.